### DISCUSSION TITLE
Sortable Properties are missing in v4

### DISCUSSION THREAD

**Discussion by PavelBrokhman** [microsoft-search/pnp-modern-search#1854] [Q&A] [answered]
Please help me with the next case scenario:
When Search results is loading it would be sorted let's say by ColumnX.
After it is loaded I would like to sort by the ColumnY. It should sort all the results not only current page. 

In v3 I could do it by configuring Sortable Properties. 

How can I do the same in v4?

**Comment by FranckyC** [comment, marked as answer]
Hi @PavelBrokhman, the sort control is not available yet in v4. Sorry for that.

**Reply by PavelBrokhman** [reply]
> Hi @PavelBrokhman, the sort control is not available yet in v4. Sorry for that.

I see. Too bad. It looks like we would need to use older version.

**Reply by FranckyC** [reply]
On the backlog, but takes time to implement. Sorry for that.
